src/test/java/example/tri/stringreverse/StringReverseTest.java

```diff
@@ -0,0 +1,13 @@
+package example.tri.stringreverse;
+
+import tri.stringreverse.StringReverse;
+
+/**
+ * Created by TriTruong on 7/12/2015.
+ */
+public class StringReverseTest {
+    public void stringreversed(){
```

Comment:
I added all the lines that you told, but I'm still not sure how to run the test.